cli/credentials.go

```diff
@@ -11,6 +11,31 @@ import (
 	ps "github.com/mitchellh/go-ps"
 )
 
+type ShellType = string
```

Comment:
Using a type alias here rather than a new type because otherwise we have to write a lot of ugly casts in get.go